nservicebus/logging/index_exception-data_core_[7.2,).partial.md

```diff
@@ -0,0 +1,3 @@
+## Additional exception data
+
+Starting from NServiceBus version 7.2, exceptions from failing message handlers might contain additional error information in the `Exception.Data` property. While the default logger exposes those information automatically, other loggers might require additional configuration.
```

Comment:
Should we make this more generic since it can contain "Data" from all parts of the pipeline and not just the handlers?

Reply:
yeah makes sense